src/lib/y2users/linux/writer.rb

```diff
@@ -271,6 +273,30 @@ def set_password_attributes(user, issues)
         log.error("Error setting password attributes for '#{user.name}' - #{e.message}")
       end
 
+      # Attributes to modify using `usermod`
+      USERMOD_ATTRS = [:home, :shell, :gecos].freeze
+
+      # Edits the user
+      #
+      # @param new_user [Y2Users::User] User containing the updated information
+      # @param old_user [Y2Users::User] Original user
+      # @param issues [Y2Issues::List] a collection for adding an issue if something goes wrong
+      def edit_user(new_user, old_user, issues)
+        usermod_changes = USERMOD_ATTRS.any? do |attr|
+          !new_user.public_send(attr).nil? &&
+            (new_user.public_send(attr) != old_user.public_send(attr))
+        end
+
+        Yast::Execute.on_target!(USERMOD, *usermod_options(new_user, old_user)) if usermod_changes
+        change_password(new_user, issues) if old_user.password != new_user.password
+        write_auth_keys(new_user, issues) if old_user.authorized_keys != new_user.authorized_keys
+      rescue Cheetah::ExecutionFailed => e
+        issues << Y2Issues::Issue.new(
+          format(_("The user '%{username}' could not be modified"), username: new_user.name)
```

Comment:
Not related to this PR, but it is quite useless message without details. What about support for details in issues and add to that stderr where is usually useful details why it failed?

Reply:
Yes, it is true. However, issues do not contain "details", so it is perhaps something to improve. Actually, you can find the details in the logs.